CONTRIBUTING.rst

```diff
@@ -168,6 +170,24 @@ or
 Use ``Resolves`` if the patch fixes the core issue which caused the bug.
 Use ``Related`` if the patch fixes an ancillary issue that is related to, but might not actually fix the bug.
 
+Release Notes
+^^^^^^^^^^^^^
+
+If you are submitting a patch that should be documented in the release notes, create a copy of the
+``docs/release-notes/template.rst`` file, modify its content and add the new file to your patch.
```

Comment:
I don't think `patch` is the correct word here. I would suggest `contribution`.

Reply:
Pull request?

Reply:
It should be clear that the file needs to be part of the changes and not attached or inserted into a pull request comment or a commit message.